### ISSUE TITLE
Property 'application.path' resolves to sa.core.dll instead of sa.shellextension.dll

### ISSUE THREAD

**Issue by end2endzone** [end2endzone/ShellAnything#124]
**Describe the bug**
Property `application.path` resolves to sa.core.dll instead of sa.shellextension.dll.
The _UserManual_ states that property `application.path` should "Matche the full path of the shell extension DLL."

**To Reproduce**
1. Load the shell extension.
2. Create the following menu:
```xml
<menu name="Debug">
  <menu name="${application.path}">
  </menu>
</menu>
```
3. Right-click on File Explorer, the shell extension's menus will be displayed.
4. Look at the name of the menu displayed under menu _Debug_.

**Expected behavior**
Property `application.path` should be set to something similar to `C:\Program Files\ShellAnything\bin\sa.shellextension.dll`.

**Screenshots**
N/A

**Environment**
 - OS: Windows 10

**Additional context**
N/A

**Comment by end2endzone** [comment]
This bug was certainly introduced when the code has migrated from a single static dll to multiple dynamic dll.

**Comment by end2endzone** [comment]
Fixed in 73d27572c9c2a5371296bccc1d341a0e4cdd08f0
